public/js/babel/schedule.js

```diff
@@ -0,0 +1,541 @@
+class schedule_frontend {
```

Comment:
The old public/schedule.js should be deleted as this replaces it

Reply:
Weird, I renamed the file in commit 460bb72, but it came back in 8b05c5d merge. Not sure how or why that happened.